docsite/rst/galaxy.rst

```diff
@@ -1,105 +1,104 @@
+discover and download them.
 
-Read the "About" page on the Galaxy site for more information.
+Learn more by viewing `the About page <https://galaxy.ansible.com/intro>`_.
 
-The ansible-galaxy command line tool
-````````````````````````````````````
+The command line tool
+`````````````````````
 
-The ansible-galaxy command has many different sub-commands for managing roles both locally and at `galaxy.ansible.com <https://galaxy.ansible.com>`_.
+The ``ansible-galaxy`` command comes bundled with Ansible, and you can use it to install roles from Galaxy or directly from a git based SCM. You can 
+also use it to create a new role, remove roles, or perform tasks on the Galaxy website. 
 
-.. note::
-
-    The search, login, import, delete, and setup commands in the Ansible 2.0 version of ansible-galaxy require access to the 
-    2.0 Beta release of the Galaxy web site available at `https://galaxy-qa.ansible.com <https://galaxy-qa.ansible.com>`_.
-
-    Use the ``--server`` option to access the beta site. For example::
-
-        $ ansible-galaxy search --server https://galaxy-qa.ansible.com mysql --author geerlingguy
-
-    Additionally, you can define a server in ansible.cfg::
-
-        [galaxy]
-        server=https://galaxy-qa.ansible.com
 
 Installing Roles
 ----------------
 
-The most obvious use of the ansible-galaxy command is downloading roles from `the Ansible Galaxy website <https://galaxy.ansible.com>`_::
+Use the ``ansible-galaxy`` command to download roles from the `Galaxy website <https://galaxy.ansible.com>`_
 
-   $ ansible-galaxy install username.rolename
+::
+
+    $ ansible-galaxy install username.role_name
 
 roles_path
 ==========
 
-You can specify a particular directory where you want the downloaded roles to be placed::
+Be aware that Ansible downloads roles to *ANSIBLE_ROLES_PATH*, which defaults to */etc/ansible/roles*. You can override this by setting the environment 
+variable in your session, defining *roles_path* in an *ansible.cfg* file, or by using the *--roles-path* option.
 
-   $ ansible-galaxy install username.role -p ~/Code/ansible_roles/
-   
-This can be useful if you have a master folder that contains ansible galaxy roles shared across several projects. The default is the roles_path configured in your ansible.cfg file (/etc/ansible/roles if not configured).
+Here's an example of using *--roles-path* to install the role into the current working directory:
 
-Installing Multiple Roles From A File
-=====================================
+::
 
-To install multiple roles, the ansible-galaxy CLI can be fed a requirements file.  All versions of ansible allow the following syntax for installing roles from the Ansible Galaxy website::
+    $ ansible-galaxy install --roles-path . geerlingguy.apache
 
-   $ ansible-galaxy install -r requirements.txt
+.. seealso::
 
-Where the requirements.txt looks like::
+   :doc:`intro_configuration`
+      All about configuration files 
```

Comment:
This file is deprecated in favor of the YAML format and should be removed in 2.3